README.md

```diff
@@ -71,6 +71,14 @@ To set it up copy the example settings from `.vscode/settings.example.json`:
 cp .vscode/settings.example.json .vscode/settings.json
 ```
 
+### Access to dev/test database
+
+When a podman compose (either dev or test one) is running you can access
+the dabase with `psql` with:
+```
+export $(cat conf/local.env conf/database_admin.env | xargs ) 2>/dev/null; ./dev/scripts/psql.sh
+```
+
```

Comment:
isn't it easier to do the following?
```bash
docker exec -it db bash
psql -d patchman -U admin
```

Reply:
That's also possible. I use the first one because it also works when you have db from openshift forwarded to your machine via `oc port-forward` ;).

Reply:
I can add both cases.

Reply:
Updated.